packages/js/plugins/ethereum/src/index.ts

```diff
@@ -411,6 +416,58 @@ export class EthereumPlugin extends Module<EthereumPluginConfig> {
   ): Promise<Connection> {
     return this._connections.getConnection(connection || this.env.connection);
   }
+
+  private _constructAbi(method: string) {
```

Comment:
minor NIT: all of these private methods that don't require the use of `this` should be put into the `utils` folder as "pure" functions. I think they'd fit nicely into the "./utils/parse.ts" module.

Reply:
100%, done :ok_hand: